common/misc.go

```diff
@@ -18,15 +20,21 @@ func HardResetHNS() error {
 	log.Debugln("Removing container networks")
 	if err := exec.Command("powershell", "Get-ContainerNetwork", "|",
 		"Remove-ContainerNetwork", "-Force").Run(); err != nil {
-		log.Debugln("Could not remove container network.")
+		log.Debugln("Could not remove nat network.")
```

Comment:
You are removing every container network existing on the machine, not just the NAT one.